target/linux/sunxi/image/Makefile

```diff
@@ -32,7 +33,6 @@ endef
 # why \x00\x00\x00\x00 for zImage-initramfs
 define Device/Default
   PROFILES := Default
-  DEVICE_VARS := SUNXI_DTS SUNXI_DTS_DIR SUNXI_UBOOT
```

Comment:
Even if this patch is not accepted eventually, we should definitely move DEVICE_VARS out of Device/Default as in this case, as this just makes no sense.